pod/podfile/views.py

```diff
@@ -432,8 +440,8 @@ def changefile(request):
     file = CustomFileModel()
     file = CustomImageModel()
 
-    if request.POST and request.is_ajax():
-        folder = get_object_or_404(UserFolder, id=request.POST["folder"])
+    if request.POST and request:
```

Comment:
Ici j'inverserai "if request and request.POST" ou alors le 2e request ne sert peut -être a rien.